ada-project-docs/wave_01.md

```diff
@@ -25,27 +25,22 @@ It's crucial for all APIs to be able to handle errors. For every required endpoi
 
 If something goes wrong, your API should return:
 - an appropriate [HTTP status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)
-- a list of errors
+- For POST and PUT requests, responses with 4XX response codes should also return an array of error messages to explain what is wrong with the request.
```

Comment:
The lack of parallel structure in the bullets really bothers me. I think a little tweaking could help, but it requires editing 26-28, which git won't let me. Suggested markdown in the following comment. It still makes it clear that there are two situations to keep in mind, and feels a bit more consistent.

Reply:
If something goes wrong...
- Your API should return an appropriate [HTTP status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).
- For POST and PUT requests, responses with 4XX response codes should also return an array of error messages to explain what is wrong with the request.